app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+    belongs_to  :user
+    has_many  :comments
+
+    validates :title,  presence: true, length: { minimum: 10 }
+    validates :category,  length: { minimum: 3 }
+    VALID_URL_REGEX = /(http|https):\/\/[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?/i 
+      validates :url, presence: true, length: { maximum: 255 },
```

Comment:
Why did we need to validate url for `post`? And in fact, why do we need this attribute at all? :)

Reply:
It has been deleted